user-management-spec.json

```diff
@@ -90,6 +90,51 @@
                 }
             }
         },
+        "/api/v1/user-profile": {
```

Comment:
Is it open API? If it needs authentication, please put up security tag

Reply:
Done